tests/modules/gatk4/variantrecalibrator/main.nf

```diff
@@ -14,7 +15,6 @@ workflow test_gatk4_variantrecalibrator {
     fasta = file(params.test_data['homo_sapiens']['genome']['genome_21_fasta'], checkIfExists: true)
     fai = file(params.test_data['homo_sapiens']['genome']['genome_21_fasta_fai'], checkIfExists: true)
     dict = file(params.test_data['homo_sapiens']['genome']['genome_21_dict'], checkIfExists: true)
-    allelespecific = false
     resources = [
```

Comment:
not very happy with the resource labels, but not sure how to approach this yet.

Reply:
@maxulysse I am not happy with this. In general, seems like this could be a more complicated use case, with somehow setting these values for different organisms/and or parameterise it. I would open a new PR for this